modules/ept-frontend/client/components/username_exists/username_exists.directive.js

```diff
@@ -0,0 +1,34 @@
+var directive = ['$timeout', '$q', '$http', function ($timeout, $q, $http) {
```

Comment:
I wonder if we could leverage a directive that we've already created for this such as the autocomplete-user-id or autocomplete-username.

I know this is a rough proof of concept, but I also noticed the username casing has to be exact with this validator.

See:
https://github.com/epochtalk/epochtalk/tree/master/modules/ept-frontend/client/components/autocomplete_user_id
or
https://github.com/epochtalk/epochtalk/tree/master/modules/ept-frontend/client/components/autocomplete_username